facebook/__init__.py

```diff
@@ -335,13 +334,14 @@ def __init__(self, result):
             # OAuth 2.0 Draft 00
             try:
                 self.message = result["error"]["message"]
+                if not self.type:
+                    self.type = result["error"].get('type', '')
```

Comment:
This is executed only if `self.type` is either an empty string or None. And as `error_code` field is returning an int that should not be the case.